index.html

```diff
@@ -10,9 +10,46 @@
 <body>
 	<div id="wrapper">
 
+		<section id="buttonHolderBG">
+			<h2 class="hidden">The Drag Zone</h2>
+			<div id="buttonHolder">
+				<div class="buttonColumn">
+					<img src="images/button0.svg" class="button" draggable="true">
+					<img src="images/button0.svg" class="button" draggable="true">
```

Comment:
Just using the first button image here. Will upload multiple buttons as they're needed.